code/controllers/subsystem/statistics.dm

```diff
@@ -0,0 +1,28 @@
+SUBSYSTEM_DEF(statistics)
+	name = "Statistics"
+	wait = 6000 // 10 minute delay between fires
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME // Only count time actually ingame to avoid logging pre-round dips
+	offline_implications = "Player count and admin count statistics will no longer be logged to the database. No immediate action is needed."
```

Comment:
This was made into a subsystem for the sake of putting the procs in one place, and incase we ever decide to start logging more statistics in the future (Gives a good branching point). 

That or ill just kill this when I do `SSblackbox`